doc/user/content/sql/functions/now_and_mz_logical_timestamp.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "now and mz_logical_timestamp functions"
+description: "In Materialize mz_logical_timestamp() acts the way now() does in most systems."
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+In Materialize, `now()` doesn't represent the system time, as it does in most systems; it represents the time with timezone when the query was executed. `mz_logical_timestamp()` comes closer to what `now()` typically indicates, since it represents the logical time at which the query executes, based on the system time defined by the system on which `materialized` is installed. Neither `now()` nor `mz_logical_timestamp()` can be used when creating views.
+
+The typical uses for `mz_logical_timestamp()` are:
+
+* Internal debugging
+* Defining [temporal filters](https://materialize.com/temporal-filters/) (`now` can also be used for this purpose)
+
+## Internal debugging
+
+`mz_logical_timestamp()` can be useful if you need to understand the time up to which data has been ingested by `materialized`. It corresponds to the timestamp column of [TAIL](/sql/tail). Generally a `CREATE SOURCE` command will cause Materialize to ingest data and produce timestamps that correspond to milliseconds since the Unix epoch.
+
+## Temporal filters
+
+You can use `mz_logical_timestamp()` to define temporal filters for materialized view, or computations for fixed windows of time. For more information, see [Temporal Filters: Enabling Windowed Queries in Materialize](https://materialize.com/temporal-filters/).
+
+## Example
+
+### Temporal filter using mz_logical_timestamp()
+
+For this example, you'll need to create a sample data source and create a materialized view from it for later reference.
+
+```sql
+--Create table
+CREATE TABLE events (
+    content text,
+    insert_ts numeric,
+    delete_ts numeric
+);
+--Create materialized view
+CREATE MATERIALIZED VIEW valid AS
+SELECT content, insert_ts, delete_ts
+FROM events
+WHERE mz_logical_timestamp() >= insert_ts
+  AND mz_logical_timestamp()  < delete_ts;
+```
+
+Next, you'll populate the table with timestamp data.
+
+```sql
+INSERT INTO events VALUES (
+    'hello',
+    mz_logical_timestamp(),
```

Comment:
I just shipped a PR that prevents use of `mz_logical_timestamp` in INSERT: https://github.com/MaterializeInc/materialize/pull/6732. You should be able to use `extract(epoch from now()) * 1000` instead.